nanopub/nanopub.py

```diff
@@ -11,6 +12,12 @@
 from nanopub import namespaces
 from nanopub.java_wrapper import JavaWrapper
 
+NANOPUB_GRLC_URLS = ["http://grlc.nanopubs.lod.labs.vu.nl/api/local/local/",
```

Comment:
Just wondering if we should have this list in a config file, or something? I believe Tobias mentioned the possibility of having new nanopub servers announcing their existence via nanopubs, so we might want a method to scan for such nanopub servers, and add to the list (probably a task for another time). But I guess this list should be in a ```known_nanopub_servers``` file, or something?

Reply:
I am in favour of just leaving it like this (which is the simplest option in my mind). If at some point we indeed have an automatic process of keeping the list of nanopub servers we can change it into a config file.

Also, I think this is a problem that should ideally be fixed on the nanopub server side. They should just have a proxy that routes traffic to one of the servers that is least loaded (load balancing). Users (like us) should not have to worry about this.